test/io/inputs.jl

```diff
@@ -61,20 +62,20 @@ end
 
 @testset "ZeroDataCube" begin
     ZeroDataCube = ADRIA.ZeroDataCube
-    
+
     dim_1_name = :timesteps
     dim_1_vals = rand(10)
-    
+
     dim_2_name = :location
     dim_2_vals = ["loc 1", "loc 2", "loc 3", "loc 4", "loc 5"]
 
     yax_res = ZeroDataCube(; T=Int, NamedTuple{(dim_1_name,)}((dim_1_vals,))...)
 
-    @test typeof(yax_res) <: YAXArray{Int, 1} || 
+    @test typeof(yax_res) <: YAXArray{Int, 1} ||
```

Comment:
@Zapiano is this test fail message correct?

Reply:
It seems correct to me (except, maybe, the instead of "received" we should say "returned")...